File: CS/ShowCrosshair/Form1.cs
using System.Windows.Forms;
using DevExpress.XtraCharts;

namespace ShowCrosshair {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        
        #region #ShowCrosshair
        private void OnChartMouseUp(object sender, MouseEventArgs e) {
            XYDiagram xyDiagram = chartControl.Diagram as XYDiagram;
            if(xyDiagram == null) return;

            xyDiagram.ShowCrosshair(e.Location);
        }
        #endregion #ShowCrosshair
    }
}

File: VB/ShowCrosshair/Form1.vb
Imports System.Windows.Forms
Imports DevExpress.XtraCharts

Namespace ShowCrosshair

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

#Region "#ShowCrosshair"
        Private Sub OnChartMouseUp(ByVal sender As Object, ByVal e As MouseEventArgs)
            Dim xyDiagram As XYDiagram = TryCast(chartControl.Diagram, XYDiagram)
            If xyDiagram Is Nothing Then Return
            xyDiagram.ShowCrosshair(e.Location)
        End Sub
#End Region  ' #ShowCrosshair
    End Class
End Namespace
